src/components/calculator.js

```diff
@@ -0,0 +1,31 @@
+export default function Calculator() {
+  return (
+    <div id="main-container">
+      <div id="calc-screen">
+        <input type="number" className="calcscr previous-op" />
+        <input type="number" className=" calcscr current-op" />
+      </div>
+      <div id="calc-buttons">
+        <button type="button" className="AC grey-btn">AC</button>
+        <button type="button" className="PlusMinus grey-btn">+/-</button>
+        <button type="button" className="Percent grey-btn">%</button>
+        <button type="button" className="Div ora-btn">÷</button>
+        <button type="button" className="Num-btn grey-btn">7</button>
+        <button type="button" className="Num-btn grey-btn">8</button>
+        <button type="button" className="Num-btn grey-btn">9</button>
+        <button type="button" className="Times ora-btn">x</button>
+        <button type="button" className="Num-btn grey-btn">4</button>
+        <button type="button" className="Num-btn grey-btn">5</button>
+        <button type="button" className="Num-btn grey-btn">6</button>
+        <button type="button" className="Minus ora-btn">-</button>
+        <button type="button" className="Num-btn grey-btn">1</button>
+        <button type="button" className="Num-btn grey-btn">2</button>
+        <button type="button" className="Num-btn grey-btn">3</button>
+        <button type="button" className="Plus ora-btn">+</button>
+        <button type="button" className="Zero span-2">0</button>
+        <button type="button" className="Dot grey-btn">.</button>
+        <button type="button" className="Equal ora-btn">=</button>
+      </div>
+    </div>
+  );
+}
```

Comment:
- [ ] Let's change the components to be class-based as stated in the requirements.

```
class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

export default Calculator;
```